gum/backend-arm/gumstalker-arm.c

```diff
@@ -3793,3 +3835,59 @@ gum_stalker_is_kuser_helper (gconstpointer address)
   return FALSE;
 #endif
 }
+
+static void
+gum_generator_context_advance_exclusive_load_offset (GumGeneratorContext * gc)
+{
+  if (gc->exclusive_load_offset == GUM_INSTRUCTION_OFFSET_NONE)
+    return;
+
+  gc->exclusive_load_offset++;
+
+  if (gc->exclusive_load_offset == 4)
+      gc->exclusive_load_offset = GUM_INSTRUCTION_OFFSET_NONE;
+}
+
+static gboolean
+gum_generator_context_is_timing_sensitive (GumGeneratorContext * gc)
```

Comment:
I tend to put getters before regular methods, so this one should go above `advance_exclusive_load_offset()`.